lib/pbench/server/auth/__init__.py

```diff
@@ -28,7 +28,11 @@ def __str__(self) -> str:
 
 
 class OpenIDTokenInvalid(Exception):
-    pass
+    def __init__(self, exc_type: Exception):
+        self.exc_type = exc_type
+
+    def __str__(self) -> str:
+        return str(f"Token invalid with exception: {self.exc_type}")
```

Comment:
I just reverted this change. If we use `isinstance` to check the type of the cause in the unit test, we don't need to touch this exception class.

Reply:
That's a good sign that using `isinstance()` is the right path.  😁

However, doing the `super()` call is "more correct" than the current code, and it enables direct (and useful) stringification of the resulting instance.  (The current implementation will inherit the `__str__()` method, but, since it doesn't call the superclass c'tor, the method will produce an empty string, which is quite unsatisfying.)